pkgs/by-name/sd/sdl3/package.nix

```
@@ -51,23 +51,30 @@
   libudevSupport ? stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid,
   sndioSupport ? false,
   testSupport ? true,
+  traySupport ? true,
   waylandSupport ? stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid,
   x11Support ? !stdenv.hostPlatform.isAndroid && !stdenv.hostPlatform.isWindows,
+  zenitySupport ? false, # sdl3 will default to dbus and fail gracefully if zenity is not available.
```

Maybe provide an alternative for those who consider using an override
```suggestion
  zenitySupport ? false, # sdl3 will default to dbus and fail gracefully if zenity is not available. sdl3 can still use zenity if available in PATH.
```